content/integrations/azure-keyvault.md

```diff
@@ -246,20 +294,164 @@ As an external application, you must register KES in [Azure Active Directory](ht
    
    For example, create a key:
 
-   ```sh
-   $ kes key create my-key-1
+   ```sh {.copy}
+   kes key create my-key-1
    ```
    
    Use the key to generate a new data encryption key:
 
+   ```sh {.copy}
+   kes key dek my-key-1
+   ```
+
+   The result resembles the following:
    ```sh
-   $ kes key dek my-key-1
    {
      plaintext : UGgcVBgyQYwxKzve7UJNV5x8aTiPJFoR+s828reNjh0=
      ciphertext: eyJhZWFkIjoiQUVTLTI1Ni1HQ00tSE1BQy1TSEEtMjU2IiwiaWQiOiIxMTc1ZjJjNDMyMjNjNjNmNjY1MDk5ZDExNmU3Yzc4NCIsIml2IjoiVHBtbHpWTDh5a2t4VVREV1RSTU5Tdz09Iiwibm9uY2UiOiJkeGl0R3A3bFB6S21rTE5HIiwiYnl0ZXMiOiJaaWdobEZrTUFuVVBWSG0wZDhSYUNBY3pnRWRsQzJqWFhCK1YxaWl2MXdnYjhBRytuTWx0Y3BGK0RtV1VoNkZaIn0=
    }
    ```
    
+## Using KES with a MinIO Server
+
+MinIO Server requires KES to set up server-side data encryption.
+
+See the [KES for MinIO instruction guide]({{< relref "/tutorials/kes-for-minio.md" >}}) for additional steps needed to use your new KES Server with a MinIO Server.
+
+
+## Configuration References
+
+The following section describes each of the Key Encryption Service (KES) configuration settings for using AWS Secrets Manager and AWS Key Management System as the root KMS for Server Side Encryption with KES.
```

Comment:
Same possible rewrite option as the one in aws-secrets-manager.md, should you choose to use that

Reply:
Looks like this file has the same info as aws-secrets-manager.md, so pretend I added the same comments. In any case, whatever you decide to do in the other location should match here.

Reply:
Had to fix throughout, because all the integration pages said they were for using AWS as the keystore! 🙃